cmd/cosign/cli/options/flags.go

```diff
@@ -46,6 +47,10 @@ type RegistryOpts struct {
 	AllowInsecure bool
 }
 
+func (co *RegistryOpts) RemoteOpts(ctx context.Context) []ociremote.Option {
```

Comment:
Let's call it `ClientOpts` or something. `remote` doesn't make sense outside the context of ggcr

Reply:
done